app/src/main/java/com/example/drcreditdev/creditCardUI/creditPage/Credit_home_page.kt

```diff
@@ -66,15 +65,15 @@ class credit_home_page : AppCompatActivity() {
             &&intent.getStringExtra("year")!=null)
         {
 
-          date.text = intent.getStringExtra("day").toString()+intent.getStringExtra("month")+"'"+intent.getStringExtra("year")
+          date.text = intent.getStringExtra("day").toString()+" "+intent.getStringExtra("month")+"'"+intent.getStringExtra("year")
         }
         else
         {
-            date.text=sharedpreferences.getString("day","12").toString()+sharedpreferences.getString("month","Apr").toString()+"'"+sharedpreferences.getString("year","22").toString()
+            date.text=sharedpreferences.getString("day","12").toString()+" "+sharedpreferences.getString("month","Apr").toString()+"'"+sharedpreferences.getString("year","22").toString()
         }
         btnGet.setOnClickListener(View.OnClickListener{
 
-            var intent = Intent(applicationContext,RefreshPage::class.java)
+            var intent = Intent(applicationContext, RefreshPage::class.java)
```

Comment:
declare intent in receiving activity, check ace credit code

Reply:
Working on it